src/utils/analytics.ts

```diff
@@ -203,6 +203,10 @@ export function recordHandledError(props: Record<string, string>) {
   recordEventWithProps('error-handled', props);
 }
 
+export function recordResearchFromEvent(eventLabel: string, properties: any) {
```

Comment:
@jackson-harris-iii Use `recordEventWithProps` directly within `ResearchFormWizard`, since there's no need to expose this method to other consumers of `analytics.ts`